src/core/vms.rs

```diff
@@ -48,7 +51,10 @@ impl ResourceTrait for Vm {
 
     fn price_per_hour(&self) -> Result<f32, ResourceError> {
         match self.price_per_hour {
-            Some(price) => Ok(price),
+            Some(price) => match self.nested_virtualization {
```

Comment:
why is nested_virtualization require to compute price_dedicated_vm_additional_per_hour ?

Reply:
You are right, it is the contrary.
nested_virtualization require to have dedicated

Reply:
https://docs.outscale.com/fr/userguide/Activer-la-virtualisation-imbriqu%C3%A9e.html